hosts.ini

```diff
@@ -0,0 +1,10 @@
+[all]
+master	ansible_host=10.60.253.143
+minion	ansible_host=10.60.253.145
```

Comment:
I think we need to try to avoid having the minions in the inventory. We can't guarantee where pods will be placed. What would we do in a 1000 node cluster?

I also think we could remove the master host (I'll explain further down), and replace it with an API host/IP. This would work in a multi-master environment.

Reply:
services are exposed on all node ports i think! https://stackoverflow.com/questions/41963433/what-does-it-mean-for-a-service-to-be-of-type-nodeport-and-have-both-port-and-t

Reply:
TIL, thanks! I guess it makes the pods relocatable.

I'm not convinced about the scalability (only so many ports available to be shared between all NodePorts) of that but hey.

It would probably be better to configure the minion IP via a variable rather than adding the host to the inventory.